visma/gui/cli.py

```diff
@@ -78,6 +81,9 @@ def commandExec(command):
     elif operation == 'differentiate':
         lhs, rhs = getLHSandRHS(tokens)
         lTokens, _, _, equationTokens, comments = differentiate(lTokens, varName)
+    else:
+        print("Please write a correct expression. \n\n")
```

Comment:
Change the warning message to "Enter valid operation\n"

Reply:
i think this would be better too.